### PR TITLE
backport: rename config `ChainName` to `ChainType`

### PR DESCRIPTION
**Pull request by bap2pecs**
https://github.com/babylonlabs-io/finality-provider/pull/24

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -45,4 +45,5 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 * [#90](https://github.com/babylonlabs-io/finality-provider/pull/90) CLI edit finality provider
 * [#91](https://github.com/babylonlabs-io/finality-provider/pull/91) Go releaser setup
   and move changelog reminder out
+* [#87](https://github.com/babylonlabs-io/finality-provider/pull/87) Rename config ChainName to ChainType
```
Comment (SebastianElvis) [inline CHANGELOG.md:48]: This is unreleased

**Review by SebastianElvis — APPROVED** [review]
Let's wait for comments from @gitferry

File: CHANGELOG.md (hunk 1 above)
Comment (bap2pecs) [reply CHANGELOG.md:48]: cc @bournezjc

Comment (bournezjc) [reply CHANGELOG.md]: It is moved to unreleased now

**Comment by bap2pecs** [comment]
cc @gitferry

File: CHANGELOG.md [hunk 2]
```diff
@@ -37,6 +37,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## Unreleased
 
+### Misc Improvements
+
+* [#87](https://github.com/babylonlabs-io/finality-provider/pull/87) Rename ChainName to ChainType
```
Comment (gitferry) [inline CHANGELOG.md:42]: missed one blank line below

**Review by gitferry — APPROVED** [review]
LGTM apart from conflicts

**Comment by bap2pecs** [comment]
fixed conflicts

**Pull request merged**